components/brave_shields/browser/https_everywhere_service.h

```diff
@@ -16,10 +16,8 @@
 #include "base/synchronization/lock.h"
 #include "brave/components/brave_shields/browser/base_brave_shields_service.h"
 #include "brave/components/brave_shields/browser/https_everywhere_recently_used_cache.h"
-
-namespace leveldb {
-class DB;
-}
+#include "brave/vendor/https-everywhere-lib-cpp/src/wrapper.hpp"
```

Comment:
the `.hpp` convention is deprecated. Even if this stays as an external repo we should follow the conventions (paths, etc...) from speedreader